packages/twenty-server/src/database/typeorm/core/migrations/common/1757364323932-1757000000000-addApplicationEntityAndRelationships.ts

```diff
@@ -0,0 +1,192 @@
+import { type MigrationInterface, type QueryRunner } from 'typeorm';
+
+export class AddApplicationEntityAndRelationships1757364323932
+  implements MigrationInterface
+{
+  name = 'AddApplicationEntityAndRelationships1757364323932';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            DO $$ BEGIN
+                CREATE TYPE "core"."application_sourcetype_enum" AS ENUM('git', 'local', 'marketplace');
```

Comment:
Not sure if we really need this. Let's just create the type and not swallow any error